### PR TITLE
docs: update the TOC

### PR DESCRIPTION
**Pull request by anupriya0703**
Signed-off-by: anupriya0703 <anupriya.gupta@mayadata.io>

### CODE REVIEW

**Comment by avishnu** [comment]
This is in "draft" mode.

**Comment by anupriya0703** [comment]
Some of the links (eg. supportability, monitoring) will not work as their corresponding PRs are under review. In order to ensure that this PR isn't merged before the others I have set it as a 'draft'.  cc @avishnu

File: SUMMARY.md [hunk 1]
```diff
@@ -1,31 +1,44 @@
...
-* [Storage Class Parameters](reference/storage-class-parameters.md)
-* [Mayastor Kubectl Plugin](reference/kubectl-plugin.md)
 * [Call-home metrics](reference/call-home.md)
 * [Tested Third Party Software](reference/tested-third-party-software.md)
 
 
+## [Basic Troubleshooting](quickstart/troubleshooting.md)
+## [FAQs](quickstart/faqs.md)
+## [Known Limitations](quickstart/known-limitations.md)
+## [Known Issues](quickstart/known-issues.md)
```
Comment (avishnu) [inline SUMMARY.md]: Extra lines below this not needed.

File: advanced-operations/kubectl-plugin.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
+# Mayastor kubectl plugin
```
Comment (avishnu) [inline advanced-operations/kubectl-plugin.md]: Is this content different from the ones in PR https://github.com/openebs/mayastor-docs/pull/86

File: SUMMARY.md [hunk 3]
```diff
@@ -1,31 +1,44 @@
...
+* [Basic Architecture](basic-architecture.md)
 
 ## Quickstart
 
-* [Scope](quickstart/scope.md)
 * [Prerequisites](quickstart/prerequisites.md)
 * [Preparing the Cluster](quickstart/preparing-the-cluster.md)
 * [Deploy Mayastor](quickstart/deploy-mayastor.md)
 * [Configure Mayastor](quickstart/configure-mayastor.md)
+  * [Storage Class Parameters](reference/storage-class-parameters.md)
```
Comment (avishnu) [inline SUMMARY.md:13]: Indentation check

File: advanced-operations/kubectl-plugin.md (hunk 2 above)
Comment (anupriya0703) [reply advanced-operations/kubectl-plugin.md]: The content is the same but this PR reflects this section as it has been moved under 'Advanced operations'. It was earlier under reference. The structural update (of entire TOC) can be seen in this PR and  content update in #86 .

File: SUMMARY.md (hunk 3 above)
Comment (anupriya0703) [reply SUMMARY.md:13]: This was done intentionally, wanted to move 'Storage class' section within 'Configure Parameter'.

**Review by avishnu — APPROVED** [review]
lgtm

**Pull request merged**